src/integration.koa.test.ts

```diff
@@ -256,3 +256,43 @@ test('introspection', async () => {
     },
   );
 });
+
+test('default parsing for POST', async () => {
+  await executeTest(
+    {
+      parse: undefined,
+      implementation: {
+        'POST /posts': (ctx) => ctx.request.body,
+      },
+    },
+    async (client) => {
+      const result = await client.post('/posts');
+
+      expect(result).toMatchObject({
+        status: 400,
+        data: "The request did not conform to the required schema: payload must have required property 'id'",
+      });
+    },
+  );
+});
+
+test('default parsing for GET', async () => {
```

Comment:
Are we missing a test for the possible override? Or is that already tested somewhere else and it wasn't surfaced in this diff?

Reply:
All of the existing tests validate custom parsers